pkg/controller/servicemesh/webhooks/mutation/controlplane_test.go

```diff
@@ -89,54 +124,43 @@ func TestNoMutation(t *testing.T) {
 // Test if the webhook defaults Version to the existing Version on an update
 func TestCreate(t *testing.T) {
 	testCases := []struct {
-		name          string
-		controlPlanes func() (runtime.Object, runtime.Object)
+		name             string
```

Comment:
as `TestCreate` and `TestNoMutation` now both follow the same pattern, maybe they should be merged into one?

Reply:
Because the test cases under both have short and not so descriptive test names, having a `TestNoMutaion` helps to understand what the tests do.